submissions/SergSenras/friends-app/script.js

```diff
@@ -0,0 +1,88 @@
+const CONTAINER = document.getElementById('container');
+const SIDEBAR = document.getElementById('sidebar');
+const SEARCH_FIELD = document.getElementById('searchField');
+const URL = 'https://randomuser.me/api/?results=40';
+
+let persons = [];
+let displayList = [];
+let searchStr = '';
+
+async function loadXMLDoc() {
```

Comment:
Could you please explain the choice of the name for this function? XML is a pretty specific name of a format.

Reply:
Well, that was the deadline )))
Fixed
```suggestion
function getPeopleData() {
```